### PR TITLE
Rename Peep properties to better distinguish Actions and Animations

### PR DESCRIPTION
**Pull request by AaronVanGeffen**
In the `Peep` struct, `Action` properties and `Animation` (was: `ActionSprite`) properties had very similar property and class names. This renames them to explicitly distinguish between `PeepActionType` and `PeepAnimationType` (was: `PeepActionSpriteType`). This translates to clearer `Update` function names as well.

This is a prerequisite for working towards staff costume objects, because the current naming conventions are driving me mad with confusion.

### CODE REVIEW

File: src/openrct2-ui/windows/Footpath.cpp [hunk 1]
```diff
@@ -817,7 +817,7 @@ namespace OpenRCT2::Ui::Windows
                         ViewportInteractionItem::LargeScenery);
 
                     auto info = GetMapCoordinatesFromPos(screenCoords, interactionFlags);
-                    if (info.SpriteType != ViewportInteractionItem::None)
+                    if (info.EntityType != ViewportInteractionItem::None)
```
Comment (Gymnasiast) [inline src/openrct2-ui/windows/Footpath.cpp:820]: Renaming it `EntityType` very heavily suggests that it takes a member of the `EntityType` enum class, which is not the case. Not to mention that `ViewportInteractionItem` has many member that don’t relate to entities at all, like `Terrain` and `Ride.`. I suggest renaming it to `type` or something like that. (Lower case, as we adopted earlier this year.)

Comment (AaronVanGeffen) [reply src/openrct2-ui/windows/Footpath.cpp:820]: Perhaps it should just be `interactionType`, per thread actual type involved.

I just happened across it in this PR due to the name conflict with the Peep property, but perhaps this is best tackled in a separate PR, anyway.

File: src/openrct2/entity/Peep.h [hunk 2]
```diff
@@ -352,13 +352,13 @@ struct Peep : EntityBase
     };
     // Normally 0, 1 for carrying sliding board on spiral slide ride, 2 for carrying lawn mower
     uint8_t SpecialSprite;
-    PeepActionSpriteType ActionSpriteType;
-    // Seems to be used like a local variable, as it's always set before calling SwitchNextActionSpriteType, which
+    PeepAnimationType AnimationType;
```
Comment (Gymnasiast) [inline src/openrct2/entity/Peep.h:355]: If we’re renaming fields, we may as well change these to be in `lowerCamelCase,` as we adopted earlier this year.

File: src/openrct2-ui/windows/Footpath.cpp (hunk 1 above)
Comment (Gymnasiast) [reply src/openrct2-ui/windows/Footpath.cpp:820]: It’s okay to tackle in here, it isn’t hard for me to review.

If you prefer `interactionType`, that’s okay with me too.

Comment (AaronVanGeffen) [reply src/openrct2-ui/windows/Footpath.cpp]: Alright, I've renamed the property to `interactionType`.

File: src/openrct2/entity/Peep.h (hunk 2 above)
Comment (AaronVanGeffen) [reply src/openrct2/entity/Peep.h:355]: It's proving to be a little painful, so I suggest we tackle the style change one entire struct at a time, indeed.

**Review by Gymnasiast — APPROVED** [review]

**Pull request merged**